transformer_engine/jax/utils.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+"""CUDA API helper"""
+import ctypes
+
+from cuda import cudart
+
+_CUDA_SUCCESS = cudart.cudaError_t.cudaSuccess
+
+
+def get_cublasLt_version():
+    """Return cuBLASLt version"""
+    libcuBLASLt = ctypes.CDLL("libcublasLt.so")
+    cublasLtGetVersion = libcuBLASLt.cublasLtGetVersion
+    cublasLtGetVersion.restype = ctypes.c_size_t
+    ver = cublasLtGetVersion()
+    return ver
+
+
+def get_cuda_version():
+    """Return CUDA version"""
+    ret, ver = cudart.cudaRuntimeGetVersion()
+    assert ret == _CUDA_SUCCESS
```

Comment:
Nitpick; all these asserts are important but could be better to wrap the call using something like `NVTE_CUDA_RETURN_CHECK` or something.

Reply:
https://github.com/NVIDIA/TransformerEngine/commit/50d9e4afd2787f45caf22d26eb13fd66ba58fbc3 solved this issue